src/main/kotlin/net/leanix/vsm/gitlab/broker/connector/application/RepositoryService.kt

```diff
@@ -41,12 +42,25 @@ class RepositoryService(
             }.getOrThrow()
     }
 
-    fun save(repository: Repository, assignment: GitLabAssignment, eventType: EventType) {
-        kotlin.runCatching {
-            repositoryProvider.save(repository, assignment, eventType)
-        }.onFailure {
-            logger.error(it) { "Failed save service" }
-        }
+    fun saveAll(repositories: List<Repository>, assignment: GitLabAssignment, eventType: EventType) {
+        repositories
+            .takeIf { it.isNotEmpty() }
```

Comment:
I think we don't need this here:
We already treat repositories not found:
https://github.com/leanix/vsm-gitlab-broker/blob/a835a59e903e2f590fe8711e290546a3872c3b34/src/main/kotlin/net/leanix/vsm/gitlab/broker/connector/application/RepositoryService.kt#L54

Reply:
done.